admin/src/components/CKEditorInput/plugins/StrapiMediaLib.js

```diff
@@ -0,0 +1,57 @@
+const Plugin = window.CKEditor5.core.Plugin;
```

Comment:
@pomek Is there a better way to import ck5 classes for dll mode than using window element? It seems a little bit dirty :)